backend/controllers/report-controller.ts

```diff
@@ -1,14 +1,41 @@
 import { Response, Request } from 'express';
 import { PrismaClient } from "@prisma/client";
+import { error } from 'console';
 
 const prisma = new PrismaClient();
 
 /**
- * @summary Add report is the main adding report page and from there
- *          we can add other reports ie - addSurfaceCoverageReport 
+ * @summary - addSurfaceCoverageReport will add to the coverage_report
```

Comment:
If you're going to have docstrings for functions, they should include parameters and return values (if any exist)